lobotomy-corp13.dme

```diff
@@ -2586,13 +2586,11 @@
 #include "code\modules\mob\living\simple_animal\abnormality\he\woodsman.dm"
 #include "code\modules\mob\living\simple_animal\abnormality\teth\beauty_beast.dm"
 #include "code\modules\mob\living\simple_animal\abnormality\teth\blood_bath.dm"
-#include "code\modules\mob\living\simple_animal\abnormality\teth\cherry_blossoms.dm"
```

Comment:
Yeowch

Reply:
Had the same issue with the ordeal pets removing abnormalities.